docs/README.md

```diff
@@ -323,7 +325,12 @@ node {
 
 ## Configuration
 
-The plugin's default behavior requires **no configuration**. If you need to change the configuration, you can use the Web UI or CasC.
+Two sources of configuration drive how the plugin works:
+
+- *AWS Java SDK configuration* (outside of Jenkins). Consult the AWS SDK documentation to find where to put the SDK configuration.
```

Comment:
This line is confusing tho, as I didnt do that type of configurations.
Maybe need to mention AWSCLI is a must for Jenkins to make calls.

Reply:
Ah, not quite. The AWS SDK is embedded in the plugin as a Java library - the Jenkins admin doesn't have to do anything extra to bring that in.

The AWS CLI is a standalone optional component. The admin may want to install it to make debugging easier from the Jenkins box, but it's not required in any way by the plugin.

Reply:
The amount of SDK config varies depending on how you do authentication with AWS...

If you have Jenkins running on EC2 and assigned an IAM role, you hardly need to configure the SDK at all, as the SDK transparently picks up the instance profile from the environment.

Meanwhile if you're running Jenkins outside of AWS, you might have a Jenkins user rather than an IAM role. Then you have more work to do. You'd need at minimum to pass the AWS_ACCCESS_KEY_ID environment variables, or if you used AWS profiles then you'd have to set up ~/.aws/config and ~/.aws/credentials.

Reply:
I thought the difference worth mentioning because I've heard from some users expecting to find the AWS SDK configuration on the Jenkins UI /configure page, but it's not there because the SDK is configured by the environment. So I'm kind of looking for a way to communicate all of this but without it being confusing 😅